doc/sandbox.md

```diff
@@ -18,7 +18,7 @@ source .venv/bin/activate
 
 Install project in sandbox mode.
 ```shell
-pip install --editable='.[io,test,develop]'
+pip install --editable='.[io,test,develop,cloud]'
```

Comment:
Maybe the project metadata gained an `all` target already, which could be used instead?
```suggestion
pip install --editable='.[all]'
```

Reply:
Here we go.

https://github.com/crate-workbench/cratedb-toolkit/blob/6d03b425afe565aea8997833ef238ef7265182fd/pyproject.toml#L100-L102

Reply:
So, I figure that instruction might be more appropriate?
```suggestion
pip install --editable='.[all,test,develop]'
```

Reply:
Implemented on behalf of e2af46e08a.